tests/ssg_test_suite/rule.py

```diff
@@ -88,8 +88,8 @@ def _apply_script(rule_dir, test_env, script):
 
         try:
             error_msg_template = (
-                f"Rule '{rule_name}' test setup script '{script}' "
-                "failed with exit code {rc}"
+                "Rule '{0}' test setup script '{1}' "
+                "failed with exit code {2}".format(rule_name, script, rc)
```

Comment:
@ggbecker @matejak 
`rc` isn't defined!

this breaks everything:

```
[jcerny@thinkpad scap-security-guide{master}]$  python3 tests/test_suite.py rule   --libvirt qemu:///system ssgts_rhel9 selinux_state 
Setting console output to log level INFO
INFO - The base image option has not been specified, choosing libvirt-based test environment.
INFO - Logging into /home/jcerny/work/git/scap-security-guide/logs/rule-custom-2022-06-03-1424/test_suite.log
INFO - xccdf_org.ssgproject.content_rule_selinux_state
WARNING - Nothing has been tested!
Traceback (most recent call last):
  File "/home/jcerny/work/git/scap-security-guide/tests/test_suite.py", line 508, in <module>
    main()
  File "/home/jcerny/work/git/scap-security-guide/tests/test_suite.py", line 504, in main
    options.func(options)
  File "/home/jcerny/work/git/scap-security-guide/tests/ssg_test_suite/rule.py", line 631, in perform_rule_check
    checker.test_target()
  File "/home/jcerny/work/git/scap-security-guide/tests/ssg_test_suite/oscap.py", line 699, in test_target
    self._test_target()
  File "/home/jcerny/work/git/scap-security-guide/tests/ssg_test_suite/rule.py", line 437, in _test_target
    self.test_rule(state, rule, scenarios_by_rule_id[rule.id])
  File "/home/jcerny/work/git/scap-security-guide/tests/ssg_test_suite/rule.py", line 337, in test_rule
    self._check_rule(
  File "/home/jcerny/work/git/scap-security-guide/tests/ssg_test_suite/rule.py", line 451, in _check_rule
    state.map_on_top(self._check_and_record_rule_scenario, args_list)
  File "/home/jcerny/work/git/scap-security-guide/tests/ssg_test_suite/test_env.py", line 26, in map_on_top
    function(* args_list[0])
  File "/home/jcerny/work/git/scap-security-guide/tests/ssg_test_suite/rule.py", line 463, in _check_and_record_rule_scenario
    self._check_rule_scenario(scenario, remote_rule_dir, rule_id, remediation_available)
  File "/home/jcerny/work/git/scap-security-guide/tests/ssg_test_suite/rule.py", line 501, in _check_rule_scenario
    if not _apply_script(
  File "/home/jcerny/work/git/scap-security-guide/tests/ssg_test_suite/rule.py", line 92, in _apply_script
    "failed with exit code {2}".format(rule_name, script, rc)
NameError: name 'rc' is not defined. Did you mean: 're'?
```

Reply:
previously the formatted string was used within a different method and only there it would evaluate the value of `rc`.

Reply:
Well, there was indeed a reason why the variable name ended with `_template` :-(

Reply:
https://github.com/ComplianceAsCode/content/pull/8888